react-instantsearch-hooks/geo-search/sandbox.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "template": "node",
```

Comment:
backend containers are much slower, could we instead use code that works on codesandbox too (without automatic runtime)?

Reply:
I thought about it and [I'm removing this sandbox config file](https://github.com/algolia/doc-code-samples/pull/399#discussion_r915904516) as there are fewer gains than I thought.